content/en/docs/gke/pipelines/authentication-sdk.md

```diff
@@ -0,0 +1,102 @@
++++
+title = "Connecting to Kubeflow Pipelines on Google Cloud using the SDK"
+description = "How to connect to different Kubeflow Pipelines installations on Google Cloud with Kubeflow Pipelines SDK?"
+weight = 20
++++
+
+This guide describes how to connect to your Kubeflow Pipelines on Google CLoud using [the Kubeflow Pipelines SDK](/docs/pipelines/sdk/sdk-overview/).
+
+## Before you begin
+
+* You need a Kubeflow Pipelines deployment on Google Cloud using one of the [installation options](/docs/pipelines/installation/overview/).
+* You need to [install Kubeflow Pipelines SDK](/docs/pipelines/sdk/install-sdk/).
+
+## How SDK connects to Kubeflow Pipelines API
+
+Kubeflow Pipelines has an API service named `ml-pipeline-ui` in the Kubernetes
+namespace you deployed it to.
+The Kubeflow Pipelines SDK can send REST api requests to this api service, but
+the SDK needs to know the hostname to connect to the API service.
+
+If the hostname can be accessed without authentication, it's very simple to
+connect to it. For example, you can use `kubectl port-forward` to access it via
```

Comment:
Can you provide an example of how to use `kubectl port-forward`?

Reply:
I put them in comments below, but it's worth making it more explicit.